### ISSUE TITLE
[training.py] Missing Required Data Files: 'ignore_list.txt' and 'train.txt'

### ISSUE THREAD

**Issue by bring-nirachornkul** [tr3e/InterGen#38]
Certainly! Here is a suggested GitHub issue that you can post to the author. This includes the topic and detailed description of the problem you're facing:

---

#### Description

Hello,

I am trying to run the `train.py` script provided in your repository. However, I encountered errors related to missing files in the `./data/interhuman_processed/` directory. Specifically, the following files are missing:

1. `ignore_list.txt`
2. `train.txt`

The absence of these files causes the script to fail with the following errors:
```
[Errno 2] No such file or directory: './data/interhuman_processed/ignore_list.txt'
[Errno 2] No such file or directory: './data/interhuman_processed/train.txt'
```

Additionally, I observed the following warning related to the checkpoint directory:
```
/home/blinkdrive/miniconda3/envs/intergen/lib/python3.8/site-packages/lightning/pytorch/callbacks/model_checkpoint.py:613: UserWarning: Checkpoint directory ./checkpoints/IG-S-8/model exists and is not empty.
  rank_zero_warn(f"Checkpoint directory {dirpath} exists and is not empty.")
```

#### Steps to Reproduce

1. Clone the repository.
2. Ensure all dependencies are installed.
3. Run the `train.py` script.

#### Expected Behavior

The script should run without any missing file errors.

#### Actual Behavior

The script fails due to missing `ignore_list.txt` and `train.txt` files.

#### Additional Information

To resolve this issue, could you please provide the missing `ignore_list.txt` and `train.txt` files? Alternatively, if these files need to be generated, could you provide the necessary instructions or scripts to create them?

#### Temporary Workaround

As a temporary workaround, I created empty placeholder files for `ignore_list.txt` and `train.txt`. However, I am not sure what data these files should contain, which might affect the training process.

#### Environment Details

- OS: Ubuntu 24.04 LTS
- Python version: Python 3.8.19
- PyTorch version: 1.13.0+cu117
- Any other relevant dependencies or versions
- absl-py==2.1.0
aiohttp==3.9.5
aiosignal==1.3.1
anyio==4.4.0
arrow==1.3.0
async-timeout==4.0.3
attrs==23.2.0
beautifulsoup4==4.12.3
blessed==1.20.0
boto3==1.34.136
botocore==1.34.136
cachetools==5.3.3
certifi==2024.6.2
charset-normalizer==3.3.2
click==8.1.7
clip @ git+https://github.com/openai/CLIP.git@dcba3cb2e2827b402d2701e7e1c7d9fed8a20ef1
croniter==1.3.15
cycler==0.12.1
dateutils==0.6.12
deepdiff==7.0.1
editor==1.6.6
exceptiongroup==1.2.1
fastapi==0.88.0
filelock==3.15.4
frozenlist==1.4.1
fsspec==2023.12.2
ftfy==6.2.0
gdown==5.2.0
google-api-core==2.19.1
google-api-python-client==2.135.0
google-auth==2.30.0
google-auth-httplib2==0.2.0
google-auth-oauthlib==1.0.0
googleapis-common-protos==1.63.2
grpcio==1.64.1
h11==0.14.0
httplib2==0.22.0
idna==3.7
importlib_metadata==8.0.0
inquirer==3.3.0
itsdangerous==2.2.0
Jinja2==3.1.4
jmespath==1.0.1
kiwisolver==1.4.5
lightning==1.9.1
lightning-cloud==0.5.70
lightning-utilities==0.11.3.post0
Markdown==3.6
markdown-it-py==3.0.0
MarkupSafe==2.1.5
matplotlib==3.2.0
mdurl==0.1.2
multidict==6.0.5
numpy==1.24.4
oauthlib==3.2.2
ordered-set==4.1.0
packaging==24.1
pillow==10.3.0
proto-plus==1.24.0
protobuf==5.27.2
psutil==6.0.0
pyasn1==0.6.0
pyasn1_modules==0.4.0
pydantic==1.10.17
Pygments==2.18.0
PyJWT==2.8.0
pyparsing==3.1.2
PySocks==1.7.1
python-dateutil==2.9.0.post0
python-multipart==0.0.9
pytz==2024.1
PyYAML==6.0.1
readchar==4.1.0
regex==2024.5.15
requests==2.32.3
requests-oauthlib==2.0.0
rich==13.7.1
rsa==4.9
runs==1.2.2
s3transfer==0.10.2
scipy==1.10.1
six==1.16.0
sniffio==1.3.1
soupsieve==2.5
starlette==0.22.0
starsessions==1.3.0
tensorboard==2.14.0
tensorboard-data-server==0.7.2
torch==1.13.0+cu117
torchaudio==0.13.0+cu117
torchmetrics==1.4.0.post0
torchvision==0.14.0+cu117
tqdm==4.66.4
traitlets==5.14.3
types-python-dateutil==2.9.0.20240316
typing_extensions==4.12.2
uritemplate==4.1.1
urllib3==1.26.19
uvicorn==0.30.1
wcwidth==0.2.13
websocket-client==1.8.0
websockets==11.0.3
Werkzeug==3.0.3
xmod==1.8.1
yacs==0.1.8
yarl==1.9.4
zipp==3.19.2

**Comment by bring-nirachornkul** [comment]
I found the way to train it already.


I will upload the file into github later.

Just Create these two files and folder interhuman_processed then drag and drop annots and motions_processed in here.

![Screenshot from 2024-07-14 23-30-31](https://github.com/user-attachments/assets/f4b80c6f-35a1-4582-bd82-716b510829a3)

**Comment by rlustc** [comment]
Hello,do you mean just create 2 empty txt files named"train.txt" and "ignore_list.txt"? If not ,could you please provide the 2 missing files? Thanks a lot!
